src/qt/locale/bitcoin_zh_TW.ts

```diff
@@ -2738,8 +2738,8 @@ Note:  Since the fee is calculated on a per-byte basis, a fee of "100 satoshis p
         <translation>複製目前的簽章到系統剪貼簿</translation>
     </message>
     <message>
-        <source>Sign the message to prove you own this Bitcoin address</source>
-        <translation>簽名這個訊息來證明這個比特幣地址是你的</translation>
+        <source>Sign the message to prove you own this Dogecoin address</source>
+        <translation>簽名這個訊息來證明這個狗狗幣地址是你的</translation>
```

Comment:
retouch the sentence.
```suggestion
        <translation>請簽署這個訊息來證明您持有這個狗狗幣地址</translation>
```